packages/spatial/src/camera/systems/CameraOrbitSystem.tsx

```diff
@@ -37,21 +37,21 @@ import {
   setComponent,
   UndefinedEntity
 } from '@ir-engine/ecs'
-import { getState, isClient } from '@ir-engine/hyperflux'
+import { getMutableState, getState, isClient } from '@ir-engine/hyperflux'
 import { TransformComponent } from '@ir-engine/spatial'
 import { CameraComponent } from '@ir-engine/spatial/src/camera/components/CameraComponent'
 import { CameraOrbitComponent } from '@ir-engine/spatial/src/camera/components/CameraOrbitComponent'
 import { Vector3_Up } from '@ir-engine/spatial/src/common/constants/MathConstants'
 import { GroupComponent } from '@ir-engine/spatial/src/renderer/components/GroupComponent'
 
+import { SelectionBoxState } from '@ir-engine/editor/src/panels/viewport/tools/SelectionBoxTool'
```

Comment:
Couple things. We can't import editor into spatial or it will be a circular dependency. I think what you are trying to do here is that if the tool is selected orbiting won't run. What you want to use in editor is InputState.setCapturingEntity(YourEntity) We can also consider other helper InputState functions. But this would remove the need to modify code in spatial like you have and then that test will stop failing.